models/preprocessing.py

```diff
@@ -11,6 +11,40 @@
 from torch.utils.data import Dataset, DataLoader
 import torch
 
+HEAD = 0
```

Comment:
Just make the dict with the label values right in the values instead of declaring all these CHEST/MRI vars?